### PR TITLE
Decode improvements

### PR DESCRIPTION
**Pull request by ferreiraandy**
> [!NOTE]
> Decode improvements

### CODE REVIEW

File: spec/site_mercado/helpers/order_parser_spec.rb [hunk 1]
```diff
@@ -79,5 +81,21 @@
...
+
+      it 'returns an decoded number' do
+        is_expected.to eql('31498-B124835579')
+      end
+    end
+
+    context 'when it is another 16 digits number' do
+      let(:encoded_id) { '3373766124852798' }
+
+      it 'returns an decoded number' do
```
Comment (thiagofm33) [inline spec/site_mercado/helpers/order_parser_spec.rb:96]:
```suggestion
      it 'returns a decoded number' do
```

File: spec/site_mercado/helpers/order_parser_spec.rb [hunk 2]
```diff
@@ -79,5 +81,21 @@
         is_expected.to eql('2681-G124488767')
       end
     end
+
+    context 'when it is a 16 digits number' do
+      let(:encoded_id) { '3149866124835579' }
+
+      it 'returns an decoded number' do
```
[the remark this answers is not in the document]
Comment (ferreiraandy) [reply spec/site_mercado/helpers/order_parser_spec.rb:88]: O grande problema do COPY/PASTE kkkk

File: spec/site_mercado/helpers/order_parser_spec.rb (hunk 1 above)
Comment (ferreiraandy) [reply spec/site_mercado/helpers/order_parser_spec.rb:96]: O grande problema do COPY/PASTE kkkk

File: spec/site_mercado/helpers/order_parser_spec.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (ferreiraandy) [reply spec/site_mercado/helpers/order_parser_spec.rb:88]: https://github.com/petlove/site_mercado/pull/92

File: spec/site_mercado/helpers/order_parser_spec.rb (hunk 1 above)
Comment (ferreiraandy) [reply spec/site_mercado/helpers/order_parser_spec.rb:96]: https://github.com/petlove/site_mercado/pull/92

**Pull request merged**